great_expectations_cloud/agent/message_service/subscriber.py

```diff
@@ -147,6 +143,14 @@ def _on_message_handler(
 
         return on_message(event_context)
 
+    @classmethod
+    def parse_event_from(cls, msg_body: bytes) -> Event:
+        try:
+            event: Event = pydantic.parse_raw_as(Event, msg_body)
+            return event
+        except (pydantic.ValidationError, JSONDecodeError):
+            return UnknownEvent()
+
```

Comment:
Ooh I like this as a separate method. Any reason why this is a `classmethod` on the Subscriber? It's kind of a builder method for an Event, I'm thinking it would be better as a `classmethod` on the Event class or module level method in the file where Event is defined. This would also help with separation of concerns (Subscriber: "Manage an open connection to an event stream." vs an event.)

Reply:
It was on Subscriber from inertia. I moved it to EventHandler as class method in 6da1c56